build.gradle

```diff
@@ -34,13 +36,18 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.json', name: 'json', version: '20180130'
+    compile group: 'org.json', name: 'json', version: '20180813'
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile group: 'it.sauronsoftware', name: 'junique', version: '1.0.4'
-    compile "de.codecentric.centerdevice:centerdevice-nsmenufx:2.1.6"
-    compile 'net.java.dev.jna:jna:5.2.0'
-
+    compile group: 'it.sauronsoftware', name: 'junique', version: '1.0.4' //for only one copy of blobsaver at a time
+    compile "de.codecentric.centerdevice:centerdevice-nsmenufx:2.1.6" //for macOS menu bar
+    compile 'net.java.dev.jna:jna:5.2.0' //for dock visibility
     testCompile 'junit:junit:4.12'
+    implementation 'org.apache.commons:commons-lang3:3.8.1'
```

Comment:
I don't want to include Apache Commons without proguard, since the library is so big. Why not just copy-paste the implementation of `StringUtils.containsIgnoreCase()` into Shared.java?

Reply:
It increased the filesize from 7.8 -> 9.9 MB, which is pretty insignificant.

Reply:
It's OK. I added it to Shared.java for now. The way Apache Commons does it is more efficient than doing `.toUpperCase()`, so I'd like to stick with it.